vlsi/Makefile

```
@@ -152,8 +157,19 @@ $(sim_files): $(SIM_FILE_REQS) $(ALL_MODS_FILELIST) | $(build_dir)
 			,\
 			echo "$(addprefix $(build_dir)/, $(notdir $(file)))" >> $@;))
 
+# copy files but ignore *.h files in *.f since vcs has +incdir+$(build_dir)
+$(fsim_files): $(SIM_FILE_REQS) $(ALL_MODS_FILELIST) | $(build_dir)
```

could not be the same from ``sim_files``?
```suggestion
$(fsim_files): $(sim_files)
```